env0/data_template.go

```diff
@@ -83,6 +83,12 @@ func dataTemplate() *schema.Resource {
 				Description: "if specified, will only retry (on destroy) if error matches specified regex",
 				Computed:    true,
 			},
+			"github_installation_id": {
+				Type:        schema.TypeInt,
+				Description: "The installation id env0 application on the repository",
```

Comment:
Please fix description (also include the word github if possible)

Reply:
b8eb7e24e86f888217bb444ede4084e39046e515